internal/pkg/aws/ecs/task.go

```diff
@@ -83,6 +88,30 @@ func (t *Task) TaskStatus() (*TaskStatus, error) {
 	}, nil
 }
 
+// ENI returns the network interface id of the running task.
+// Every Fargate task is provided an ENI by default (https://docs.aws.amazon.com/AmazonECS/latest/userguide/fargate-task-networking.html).
+func (t *Task) ENI() (string, error) {
+	attachmentENI := &ecs.Attachment{}
```

Comment:
Can we do sth like this so that we don't need `hasENIAttachment`?
```suggestion
	var attachmentENI *ecs.Attachment{}
```